### PR TITLE
1.5.0.Beta2 announcement

### PR DESCRIPTION
**Pull request by Naros**
@gunnarmorling @jpechane Could you please review and let me know if any changes are needed?  I'll merge this in 2 hours at, 4:30pm UTC.

### CODE REVIEW

File: _posts/2021-03-15-debezium-1-5-beta2-released.adoc [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+        "generated": false
+      }, {
+        "name": "aa",
+        "jdbcType": 4,
+        "nativeType": 23,
+        "typeName": "int4",
+        "typeExpression": "int4",
+        "charsetName": null,
+        "length": 10,
+        "scale", 0,
```
Comment (jpechane) [inline _posts/2021-03-15-debezium-1-5-beta2-released.adoc:105]: Should be colon, not comma, also see above

File: _posts/2021-03-15-debezium-1-5-beta2-released.adoc [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  }]
+}
+```
+
+With this JSON payload, the signal would be inserted as:
+
+```sql
+INSERT INTO DEBEZIUM_SIGNALS (ID, TYPE, DATA) VALUES ('1', 'schema-changes', <json-payload-string>);
+```
+
```
Comment (gunnarmorling) [inline _posts/2021-03-15-debezium-1-5-beta2-released.adoc:121]: A sentence is needed why one would want to use this: Corner cases around schema changes for Postgres (PK type changes, changes to allowed values of enum columns), schema changes for Oracle (in absence of a DDL parser).

Also let's point out it's an incubating feature for the time being.

File: _posts/2021-03-15-debezium-1-5-beta2-released.adoc [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+== Other Features and Fixes
```
Comment (gunnarmorling) [inline _posts/2021-03-15-debezium-1-5-beta2-released.adoc:122]: Nothing else of interest to mention, out of 56 issues?

**Review by gunnarmorling — COMMENTED** [review]
Thanks a lot, a few comments inline.

File: _posts/2021-03-15-debezium-1-5-beta2-released.adoc (hunk 3 above)
Comment (Naros) [reply _posts/2021-03-15-debezium-1-5-beta2-released.adoc:134]: I added a few more things that seemed noteworthy.  If there is something specific you'd like me to add let me know as most of the changes were either documentation, bugfixes, or general polish afaict.

File: _posts/2021-03-15-debezium-1-5-beta2-released.adoc [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+* The Debezium connector for Oracle now uses the LogMiner-based capturing implementation by default.
+In order to use the XStream-based implementation, the `database.connection.adapter` option must be explicitly set to `xstream` (https://issues.redhat.com/browse/DBZ-3241[DBZ-3241]).
+
+* In an earlier release of Debezium 1.5, the Oracle connector began to emit `NUMBER(1)` data types as `BOOLEAN`.
+Rather than this conversion be implicitly done by the connector, this behavior has been moved to an OOTB converter, `NumberOneToBooleanConverter`,
+that can be used as needed (https://issues.redhat.com/browse/DBZ-3208[DBZ-3208]).
+
+* System generated index-organized tables (tables that begin with `SYS_IOT_OVER`) are ignored by the Oracle connector (https://issues.redhat.com/browse/DBZ-3036[DBZ-3036])
+
+* Debezium server's kenesis sink can be configured with an endpoint by specifying `debezium.sink.kenesis.endpoint` (https://www.redhat.com/browse/DBZ-3246[DBZ-3246]).
```
Comment (gunnarmorling) [inline _posts/2021-03-15-debezium-1-5-beta2-released.adoc:147]:
```suggestion
* Debezium Server's sink for AWS Kinesis can be configured with an endpoint by specifying `debezium.sink.kinesis.endpoint` (https://www.redhat.com/browse/DBZ-3246[DBZ-3246]).
```

**Review by gunnarmorling — APPROVED** [review]

**Pull request merged**